common/components/common/positions/AboutPositionEntry.jsx

```diff
@@ -44,15 +46,34 @@ class AboutPositionEntry extends React.PureComponent<Props> {
   }
 
   _renderApplyButton(): ?React$Node {
+
+    let ApplyButton;
```

Comment:
```suggestion
    let applyButton;
```
Use camelCase to avoid confusing variable with Component.